libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -13,8 +13,18 @@ async function getKey(product) {
   return keyMatch[0]?.key;
 }
 
+async function getECID() {
+  let ecid = null;
+  if (window.alloy) {
+    await window.alloy('getIdentity').then((data) => {
```

Comment:
you should have a catch here in case the promise does not resolve

Reply:
Yes sure, added this!